inspirehep/modules/records/jsonschemas/records/elements/urls.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
JSON Schema needs to start with:
```javascript
"$schema": "http://json-schema.org/schema#"
```

See an example [here](https://github.com/tomaszgy/inspire-next/blob/clean_mappings/inspirehep/modules/records/jsonschemas/records/json_reference.json)